app/models/casino/service_ticket.rb

```diff
@@ -7,25 +7,27 @@ class CASino::ServiceTicket < ActiveRecord::Base
   has_many :proxy_granting_tickets, as: :granter, dependent: :destroy
 
   def self.cleanup_unconsumed
-    self.delete_all(['created_at < ? AND consumed = ?', CASino.config.service_ticket[:lifetime_unconsumed].seconds.ago, false])
+    where('created_at < ? AND consumed = ?', CASino.config.service_ticket[:lifetime_unconsumed].seconds.ago, false).delete_all
   end
 
   def self.cleanup_consumed
-    self.destroy_all(['(ticket_granting_ticket_id IS NULL OR created_at < ?) AND consumed = ?', CASino.config.service_ticket[:lifetime_consumed].seconds.ago, true])
+    where('(ticket_granting_ticket_id IS NULL OR created_at < ?) AND consumed = ?', CASino.config.service_ticket[:lifetime_consumed].seconds.ago, true).destroy_all
   end
 
   def self.cleanup_consumed_hard
-    self.delete_all(['created_at < ? AND consumed = ?', (CASino.config.service_ticket[:lifetime_consumed].seconds * 2).ago, true])
+    where('created_at < ? AND consumed = ?', (CASino.config.service_ticket[:lifetime_consumed].seconds * 2).ago, true).delete_all
   end
 
 
   def service=(service)
-    normalized_encoded_service = Addressable::URI.parse(service).normalize.to_str
+    normalized_encoded_service = Addressable::URI.parse(service)&.normalize&.to_str || ''
```

Comment:
should be double quotes

Reply:
if we'd enable rubocop here, I think there would be more refactoring to do

Reply:
rubocop is just a tool that supervises our coding guidelines, and we decided to use double quotes https://github.com/konvenit/guidelines/blob/master/ruby.md#consistent-string-literals